metricflow/sql_clients/databricks.py

```diff
@@ -18,6 +18,9 @@
 logger = logging.getLogger(__name__)
 
 HTTP_PATH_KEY = "httppath="
+HTTP_PATH_RENAME_KEY = "httppathrename="
```

Comment:
Does this only trigger on renames, or other alter table commands? If not, we should name this key something more generic, and maybe broaden out our match pattern below.

If we don't know let's leave it this way - since this isn't part of the public interface for SqlClient we should be able to change it easily later.

Reply:
Only renames!